flake.nix

```diff
@@ -0,0 +1,79 @@
+{
+  description = "Slack library for Haskell";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  nixConfig.allow-import-from-derivation = true; # cabal2nix uses IFD
+
+  outputs = { self, nixpkgs, flake-utils }:
+    let
+      ghcVer = "ghc924";
```

Comment:
Don't we want this to be GHC 9.4? Or am I confused about something?

Reply:
This is for dev purposes; HLS kinda sucks on 9.4 at present. I just used the flake template I had, but more versions could also be added in the future.

Reply:
Also 9.4 is not yet fixed up in nixpkgs, which w1ggles is working on fixing https://github.com/NixOS/nixpkgs/pull/191618 :)